openmeter/meter/parse.go

```diff
@@ -0,0 +1,93 @@
+package meter
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/cloudevents/sdk-go/v2/event"
+	"github.com/oliveagle/jsonpath"
+
+	"github.com/openmeterio/openmeter/pkg/models"
+)
+
+// ParseEvent validates and parses an event against a meter.
+func ParseEvent(meter Meter, ev event.Event) (*float64, *string, map[string]string, error) {
```

Comment:
Why is validate being replaced by parsing?

Reply:
The new connector required to know value. It was always parsing, just before didn't return the parsed value. This file looks new but it's mostly renamed.  I removed the new connector from the PR but would like to keep parsing this way as it helps testing.